### ISSUE TITLE
Firefox new version - no website working anymore

### ISSUE THREAD

**Issue by rohrbachger** [Thinstation/thinstation#630]
I have upgraded to firefox to 85.0.2
The installation works in general, but no website is working anymore.
No access to mozillas web pages.
General network connection is fine, I can work with Horizon View, so nothing is stopping the internet access.
I have tested, disabling ipv6 but still no success, also proxy setting I checked.
I installed Chrome, this works , so i show the network config is okay.

Can someone help with a hint where to search.

**Comment by Doncuppjr** [comment]
Likely need to update nss and nspr.

**Comment by rohrbachger** [comment]
I sure you are right, there were changes in nss.
I have uninstalled the nss package, and it was installed automatically again. 
firefox dependencies.
But still the same problem. How does it works to update?
I found under ports/opt/nss 3.57-1.pkg.tar.xz, an old version
An under nspr#4.29-1.pkg.tar.xz 
Do I have just to upgrade this to new versions, and then uninstall again the nss package to get the new version?
Or is there another more easy way, to get the port from crux?

**Comment by Doncuppjr** [comment]
I updated nss to 3.60.


Sent from Yahoo Mail for iPhone


On Sunday, February 14, 2021, 11:03 PM, rohrbachger <notifications@github.com> wrote:




I sure you are right, there were changes in nss.
I have uninstalled the nss package, and it was installed automatically again.
firefox dependencies.
But still the same problem. How does it works to update?
I found under ports/opt/nss 3.57-1.pkg.tar.xz, an old version
An under nspr#4.29-1.pkg.tar.xz
Do I have just to upgrade this to new versions, and then uninstall again the nss package to get the new version?
Or is there another more easy way, to get the port from crux?
